### DISCUSSION TITLE
Options for Livolo in combination with other brands

### DISCUSSION THREAD

**Discussion by definable** [Koenkk/zigbee2mqtt#8369] [General]
I just installed zigbee2mqtt via Docker compose, with the intention to control a number of Livolo touch switches. 
Had to find out a couple of things, but it did not take much time to get it up and running and to control a switch. 
So well done and thanks, for everybody involved !

Now my question. For Livolo I am bound to the PAN ID configuration and the channel number.  Suppose I want to use other Zigbee devices as well, what would be the best approach ? 

- Is it possible to support 2 networks with different PAN ID from a single coordinator ?
- Do I need to add a second USB key with a second zigbee2MQTT instance ?

Any suggestions and/or experiences  related to Livolo in combination with other brands ?

Thanks in advance,

**Comment by avlisiur78** [comment]
Hi,

You can have in the same network, as the Livolo devices, other devices from other brands, the only limitation that you will have is the lost of possibility of changing the channel and PAN ID.
